server/Resources/Resources.resx

```diff
@@ -4911,4 +4911,7 @@ Set to "External URL" to use an API definition that is hosted elsewhere.</value>
   <data name="readOnlyJava" xml:space="preserve">
     <value>Editing functions in the portal is not supported for Java Function Apps. Please use Maven or VS Code for development.</value>
   </data>
+  <data name="error_deploymentCenterExists" xml:space="preserve">
+    <value>Deployment center is already setup on your web site. Kindly naviagte to web app and disconnect before proceeding. Deployment slots are considered part of the web app itself.</value>
```

Comment:
Text suggestion: Deployment is already configured through Deployment Center on your web app. To reconfigure, kindly disconnect the existing one before proceeding further. Note that only one deployment configuration is allowed through Deployment Center for a web app, in spite of the number of slots.